### DISCUSSION TITLE
No New Faces in People>NEW

### DISCUSSION THREAD

**Discussion by artincube** [photoprism/photoprism#3535] [Q&A]
It stays empty, doesn't repopulate with numerous unnamed faces from all images found in face:new (clicking on "Show all new faces").
What is supposed to be the behavior?
I was expecting this to repopulate as I was tagging them, moving new to yes, making space for other new ones.
I think it did for a while.
It stopped. I am frustrated because it takes much longer to do it through the search page and editing images metadata.
Am I missing something?

**Comment by lastzero** [comment]
https://docs.photoprism.app/user-guide/organize/people/#why-doesnt-the-new-faces-page-show-all-faces
